README.md

```diff
@@ -1,2 +1,44 @@
-# rss-notification-service
-A prototype for RSS notification emails
+# RSS Notification Service
```

Comment:
Can you add the standard contribution and license sections from Augustus
https://github.com/ukparliament/augustus